### PR TITLE
Document server access enhanced recording option `root_path`

### PR DESCRIPTION
**Pull request by gabrielcorado**
Documents new `ssh_service.enhanced_recording.root_path` option added by #38066.

### CODE REVIEW

**Comment by zmb3** [comment]
Should we mention this anywhere outside of the inline comment in the configuration?

File: docs/pages/includes/config-reference/ssh-service.yaml [hunk 1]
```diff
@@ -54,6 +54,10 @@ ssh_service:
        # /cgroup2.
        cgroup_path: /cgroup2
 
+       # Optional: Controls the path inside cgroupv2 hierarchy where Teleport
+       # cgroups will be placed. Default value: /teleport
```
Comment (ptgott) [inline docs/pages/includes/config-reference/ssh-service.yaml:58]: When would a user want to change this?

File: docs/pages/server-access/guides/bpf-session-recording.mdx [hunk 2]
```diff
@@ -163,8 +163,22 @@ ssh_service:
...
+    # Optional: Controls the path inside cgroupv2 hierarchy where Teleport
+    # cgroups will be placed. Default value: /teleport
+    root_path: /teleport
 ```
 
+<Details title="Isolate session recordings">
+
+If you operate multiple Teleport instances on the same system, customize the
+`root_path` configuration to change the base cgroup slice path for session
+recordings resources. This ensures security isolation between recordings from
```
Comment (ptgott) [inline docs/pages/server-access/guides/bpf-session-recording.mdx:176]: I'm not sure what "session recordings resources" means. Should this be "session recordings"?

**Review by ptgott — APPROVED** [review]
Approved with a comment, assuming Zac's feedback gets applied.

**Comment by ptgott** [comment]
@gabrielcorado Are we backporting this PR?

File: docs/pages/server-access/guides/bpf-session-recording.mdx [hunk 3]
```diff
@@ -163,8 +163,22 @@ ssh_service:
     # Optional: Controls where cgroupv2 hierarchy is mounted. Default value:
     # /cgroup2.
     cgroup_path: /cgroup2
+
+    # Optional: Controls the path inside cgroupv2 hierarchy where Teleport
+    # cgroups will be placed. Default value: /teleport
+    root_path: /teleport
 ```
 
+<Details title="Isolate session recordings">
```
[the remark this answers is not in the document]
Comment (gabrielcorado) [reply docs/pages/server-access/guides/bpf-session-recording.mdx]: I've updated it, and the content to relate to those resources as "system resources". This aligns with how cgroup describe its hierarchy:

> [cgroup is a mechanism to organize processes hierarchically and distribute system resources along the hierarchy in a controlled and configurable manner.](https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html#what-is-cgroup)

File: docs/pages/server-access/guides/bpf-session-recording.mdx (hunk 2 above)
Comment (gabrielcorado) [reply docs/pages/server-access/guides/bpf-session-recording.mdx]: I've updated it to match the title. [More info at my comment above.](https://github.com/gravitational/teleport/pull/38399#discussion_r1508195453)

**Review by greedy52 — APPROVED** [review]
lint seems still failing

**Pull request merged**